src/envo/envo-edit.owl

```diff
@@ -41989,8 +42006,147 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501354> <http://purl.obolibrar
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501361> "wheel"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501361> <http://purl.obolibrary.org/obo/ENVO_03501357>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501362> (belt conveyor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://instrumentationtools.com/conveyor/>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501362> "A conveyor system which uses a conveyer belt component to convey material."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501362> <https://orcid.org/0000-0002-2738-824X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501362> "2021-12-09T00:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/ENVO_03501362> "conveyor belt"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501362> "belt conveyor"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501362> <http://purl.obolibrary.org/obo/ENVO_03501355>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501362> <http://purl.obolibrary.org/obo/ENVO_03501356>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501362> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03501283>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501363> (conveyor belt component)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://instrumentationtools.com/conveyor/>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501363> "A manufactured product which is 1) an endless belt of flexible material stretched between two drums; 2) conveys material between drum locations."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501363> <https://orcid.org/0000-0002-2738-824X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501363> "2021-12-09T00:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/ENVO_03501363> "conveyor belt"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_03501363> <https://www.conveyorsystemsltd.co.uk/news/168/126/Roller-Conveyor-vs-Belt-Conveyor.htm>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501363> "conveyor belt component"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501363> <http://purl.obolibrary.org/obo/ENVO_00003074>)
```

Comment:
you have no connection between conveyer belt component and your new term conveyor belt

we don't need to over-axiomatize things, but we should aim for reasonable completeness for part-of axioms

(this can be addressed after merging)